Doc/library/typing.rst

```diff
@@ -500,6 +504,8 @@ The module defines the following classes, functions and decorators:
 
     A generic version of :class:`collections.abc.Reversible`.
 
+   .. versionadded:: 3.5.2
```

Comment:
@ilevkivskyi I mentioned `Reversible` in the tracker due to it's listing in [what's new for Python 3.5.2](https://docs.python.org/3.5/whatsnew/changelog.html#id12). [Here's the change you're talking about too](https://github.com/python/cpython/commit/bd5b9a074241c6146bdcacef9a1e84f48c7c8d8a). I guess that either the version needs to get dropped for `Reversible` or the What's new entry changed, not sure about it though :-)